src/App.jsx

```diff
@@ -1,41 +1,140 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import './App.scss';
 import './styles/general.scss';
+
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+// eslint-disable-next-line max-len
+import { getPosts, getUserPosts, getPostDetails, getPostComments, deletComment, createPost } from './api/post';
```

Comment:
```suggestion
import { 
  getPosts, getUserPosts, getPostDetails, 
  getPostComments, deletComment, 
  createPost,
} from './api/post';
```
Зачем отключать линтер, если можно перенести на новую строку?